Jenkinsfile

```diff
@@ -385,7 +385,7 @@ pipeline {
                     }
 
                     withXTAG(["usb_audio_mc_xcai_dut", "usb_audio_mc_xcai_harness"]) { xtagIds ->
-                      sh "pytest -v --level ${params.TEST_LEVEL} --junitxml=pytest_result_windows11.xml \
+                      sh "pytest -s -v --level ${params.TEST_LEVEL} --junitxml=pytest_result_windows11.xml \
```

Comment:
not sure if you meant to leave -s here or if it was just for debugging. Ignore if intentional.

Reply:
It was kind of intentional - You only look at the log if something has failed and there is precious little debug info so I thought it was helpful. Happy to remove if others prefer a cleaner output.